client/src/navigators/Tabs.js

```diff
@@ -52,6 +53,7 @@ export default () => {
         component={ProfileStacks}
         options={{ tabBarLabel: 'Profile', tabBarIcon: ({ width }) => <ProfileIcon width={width} /> }}
       />
+      <Tab.Screen name="NotificationTab" component={NotificationStacks} />
```

Comment:
import Notification screen

```suggestion
      <Tab.Screen name="Notification" component={Notification} />
```